Blazor_Foundation_6_Test_Shared/Documentation/Containers/Callout.razor

```diff
@@ -1,6 +1,119 @@
 ﻿@page "/callout";
 
-<h3>Callout</h3>
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+@using OpenCodeDev.Blazor.Foundation.Components.Controls;
+@using OpenCodeDev.Blazor.Foundation.Components.Containers;
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Callout</h2>
+        <p>Callouts combine panels and alerts from Foundation 5 into one generic container component.</p>
+
+        <h2>Basic</h2>
+        <p>A callout is just an element with a <Code>.callout</Code> class applied. You can put any kind of content inside.</p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PENhbGxvdXQ+CiAgICA8aDU+VGhpcyBpcyBhIGNhbGxvdXQuPC9oNT4KICAgIDxwPkl0IGhhcyBhbiBlYXN5IHRvIG92ZXJyaWRlIHZpc3VhbCBzdHlsZSwgYW5kIGlzIGFwcHJvcHJpYXRlbHkgc3ViZHVlZC48L3A+CiAgICA8TmF2TGluayBocmVmPSIjIj5JdCdzIGRhbmdlcm91cyB0byBnbyBhbG9uZSwgdGFrZSB0aGlzLjwvTmF2TGluaz4KPC9DYWxsb3V0Pg==" />
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+            <h5>This is a callout.</h5>
+            <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            <NavLink href="#">It's dangerous to go alone, take this.</NavLink>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+        <hr />
+
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Coloring</h2>
+        <p>
+            Callouts can be colored using the <Code>.secondary</Code>, <Code>.primary</Code>, <Code>.success</Code>, <Code>.warning</Code>, or <Code>.alert</Code> classes. Links inside the callout will be tinted to match the color of the callout.
```

Comment:
Same here look at Button Group its a very good template to understand the way. Button Group->Coloring